.monorepolint.config.ts

```diff
@@ -163,63 +129,28 @@ module.exports = {
       {
         options: {
           scripts: {
-            build: "npm-run-all build:*",
-            "build:js": "tsc",
-            "build:es":
-              'tsc --outDir dist/es --module esnext --declaration false && echo \'{"type":"module"}\' > dist/es/package.json',
+            build: "tsup --config ../../tsup.config.ts",
           },
         },
-        includePackages: TS_PACKAGES,
+        includePackages: [...JS_PACKAGES, ...TS_PACKAGES],
       },
       {
         options: {
           scripts: {
             build:
-              'rollup -c ../../rollup.config.js && echo \'{"type":"module"}\' > dist/es/package.json',
-          },
-        },
-        includePackages: JS_PACKAGES,
-      },
-      {
-        options: {
-          scripts: {
-            build:
-              'rollup -c rollup.config.js && echo \'{"type":"module"}\' > dist/es/package.json',
+              "tsup --config ../../tsup.config.ts && rollup -c ./rollup.config.js",
           },
         },
         includePackages: [MAIN_PACKAGE],
       },
       {
         options: {
           scripts: {
-            "test:tape": "node -r esm test.js",
-          },
-        },
-        includePackages: JS_TAPE_PACKAGES,
-      },
-      {
-        options: {
-          scripts: {
-            "test:tape": "ts-node -r esm test.js",
+            bench: "tsx bench.js",
```

Comment:
Everything is handled by tsx now. I tried to get this to work with `-r esm`, but I got really weird errors.